### PR TITLE
Holosigns: now In Use, less gaming more cohesion, a ton more of QoL changes

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- Comments out Vomitol and its recipe, virtually useless chem that was scrambling Inaprovaline's recipe
- Minor grammatical fixes to xenoarch manual, adds it (#1057 ) on a table near the Suspension Field Generators
- Makes Suspension Field Generators area and lockers more cohesive to avoid bumping near the entrance of the mines
- Adds a duffelbag and membrane labcoat to xenoarch lockers, as requested
- Orderlies now get a very basic "helmet" instead of surgical caps, and their lockers contain a specialist labcoat each.
- Fixed improper message on soap examine text to know if it's wet enough or not (`"the the [src] of soap looks wet"` changed to `"\the bar of [src] looks wet"`)
- Adds toggleable Holosigns to the Bar rooms and Surgery room, to make sure people know if they're in use (privacy!)
- Adds a bolt door button for the southmost bar room, **Room B**
- Fixes door bolt button not properly bolting the door to Secure Storage (leaves access intact to both Adepts and the GM)
- Adds a tint button window to Guild Master's office, correctly labels electrochromic windows to him rather than be the same as CRO office, and replaces the glass airlock for a normal one or the tint would be moot.
- Adds proper access to certain maintenance doors that lead into sensitive areas not everybody should barge into. **ADDENDUM:** Apparently the Soteria one got fixed while I did this PR but I still changed two others (Marshals and Atmos) that were free access and totally shouldn't have.
- Changes recipe for table salt to include water for chemical cohesion (hydrocloric acid + water + sodium)
- Dylovene now removes Carpotoxin from the blood rather than Blattedin, given to how it conflicted working with Cht'mant healing and how it was moot to remove Blattedin from a human's blood since it's innocuous for humans
- Ossissine now puts bones back in place only on overdose, which is dangerous since it causes cloneloss damage. This is to avoid having a one-wonder chem and require actual surgery. It should also metabolize way faster given how it's an instant knockdown chem, to make it fairer and more cohesive on the situations it's used (combat bone fixing, non-ideal medical situations, etc)
- Hopefully fixes the chemical scrambling of Chemical Dispensers when upgraded, as well as changing the chems it gives per upgrades and via hacking (erased duped cryptobiolin). This also applies for the Industrial Chem Dispenser, now more oriented to the production of grenades, explosives, surfactant, etc...
- Eliminates dupe soda machine on bar, replaced by barman's recipes book, places the chef's recipe book on kitchen, both versions linked to the Wiki page.
- Centers the jukebox, moves the ATM closer to the Bar (switching places with the newscaster)
- Adds checks for correct gendering of colony mascots (namely Ian, Lisa and Runtime)
- Removes eggplant from Stew recipe. Eggplants do not go in stew, we need onions. Why has nobody coded onions?!
- Ruins everything
- Why do I keep coding when sleep deprived
- I'm slowly going insane
- Make it stop

### CODE REVIEW

File: code/modules/reagents/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -22,23 +21,23 @@
...
+		"lithium", "mercury", "phosphorus",
+		"potassium", "radium", "sacid",
+		"silicon", "sodium", "sugar",
+		"sulfur", "tungsten", "water",
 	)
 
 	var/list/level1 = list("oil", "cryptobiolin")
 	var/list/level2 = list("toxin", "sodiumchloride")
-	var/list/level3 = list("potassium_chloride", "cryptobiolin")
+	var/list/level3 = list("mutagen")
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/machinery/chem_dispenser.dm:37]:
```suggestion
	var/list/level3 = list("mutagen", "potassium_chloride", "cryptobiolin") 
```
They unlock for a reason

File: code/modules/reagents/machinery/chem_dispenser.dm [hunk 2]
```diff
@@ -353,11 +352,14 @@
...
-		"iron","radium","sacid",
-		"hclacid","silicon","tungsten"
+		"copper","ethanol","hclacid",
+		"hydrazine","iron","phosphorus",
+		"potassium", "radium","sacid",
+		"silicon","sulfur","tungsten"
 	)
 
 	level1 = list("oil", "cryptobiolin", "sterilizine")
+	level2 = list("foaming_agent", "plasticide", "instant_ice")
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/machinery/chem_dispenser.dm:362]:
```suggestion
	level2 = list("foaming_agent", "plasticide", "instant_ice", "toxin", "sodiumchloride")
```
you likely didnt know, but ye this needs to be added to the list for consistancy

File: code/modules/reagents/machinery/chem_dispenser.dm [hunk 3]
```diff
@@ -353,11 +352,14 @@
...
+	level3 = list("pacid") //Now it has chems for what it's supposed to do, and worth upgrading past Picos
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/machinery/chem_dispenser.dm:363]:
```suggestion
	level3 = list("pacid", "mutagen", "potassium_chloride", "cryptobiolin") //Now it has chems for what it's supposed to do, and worth upgrading past Picos
```
as above

File: code/modules/reagents/machinery/chem_dispenser.dm [hunk 4]
```diff
@@ -353,11 +352,14 @@
...
 
-	hacked_reagents = list("cleaner","silicate","coolant") //So we have a reason to keep you
+	hacked_reagents = list("surfactant","silicate","coolant") //So we have a reason to keep you
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/machinery/chem_dispenser.dm:365]:
```suggestion
	hacked_reagents = list("cleaner","surfactant","silicate","coolant") //So we have a reason to keep you
```

File: code/modules/reagents/reagents/medicine.dm [hunk 5]
```diff
@@ -128,7 +128,7 @@
 	M.adjustToxLoss(-((0.4 + (M.getToxLoss() * 0.05)) * effect_multiplier))
 	M.add_chemical_effect(CE_ANTITOX, 1)
 	holder.remove_reagent("pararein", 0.8 * effect_multiplier)
-	holder.remove_reagent("blattedin", 0.4 * effect_multiplier)
+	holder.remove_reagent("carpotoxin", 0.4 * effect_multiplier) // Gonna be good for fish recipes
 	holder.remove_reagent("toxin", 0.4 * effect_multiplier)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/medicine.dm:132]:
```suggestion
	holder.remove_reagent("toxin", 0.4 * effect_multiplier)
	holder.remove_reagent("blattedin", 0.4 * effect_multiplier)
```

**Comment by Trilbyspaceclone** [comment]
Mostly just the chem dispender tweaks needs to be fixed rest seem ok

File: code/modules/reagents/machinery/chem_dispenser.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/modules/reagents/machinery/chem_dispenser.dm:37]: That cryptobiolin is a dupe, it needs to go.

File: code/modules/reagents/machinery/chem_dispenser.dm [hunk 6]
```diff
@@ -22,23 +21,23 @@
...
 	)
 
 	var/list/level1 = list("oil", "cryptobiolin")
 	var/list/level2 = list("toxin", "sodiumchloride")
-	var/list/level3 = list("potassium_chloride", "cryptobiolin")
+	var/list/level3 = list("mutagen")
 	var/list/level4 = list("inaprovaline")
 
-	var/list/hacked_reagents = list("cleaner") //Basic stuff
+	var/list/hacked_reagents = list("mindbreaker") //USEFUL stuff
```
[the remark this answers is not in the document]
Comment (ThePainkiller) [reply code/modules/reagents/machinery/chem_dispenser.dm:40]: Eh, why not, if it's okay with you...just didn't want to create a wall of text by adding too much

File: code/modules/reagents/machinery/chem_dispenser.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/modules/reagents/machinery/chem_dispenser.dm:362]: I figured it's an industrial dispenser, for industrial-oriented work (hence plasticide, foaming agent, etc), how does salt fit on it?

File: code/modules/reagents/machinery/chem_dispenser.dm (hunk 3 above)
Comment (ThePainkiller) [reply code/modules/reagents/machinery/chem_dispenser.dm:363]: Again, it's an **industrial** themed chem dispenser, why would you need to make a lethal injection chemical and antibiotics with it...?

File: code/modules/reagents/reagents/medicine.dm (hunk 5 above)
Comment (ThePainkiller) [reply code/modules/reagents/reagents/medicine.dm:132]: Please refer to Blattedin code and how its `affect_blood` overrides its toxin parent, it's completely innocuous for humans, only affects roaches for revival and Cht'mant for healing, and this helps applying Dylovene to Cht'mant as well as Blattedin for better cohesive healing without one stepping over the other. It makes more sense for Dylovene to remove an actual toxic component that just mere roach farts on the blood, and this change helps Cht'mant as well, who are well a pain in the tail to fix already.

**Pull request merged**